pages/learn/deploy/delta.md

```diff
@@ -15,7 +15,7 @@ These binary deltas save on the amount of data needed to be downloaded, reduce t
 
 ## Enabling delta updates
 
-For any devices running {{ $names.os.lower }} >= 2.47.1, the delta update behavior is enabled by default. For devices running {{ $names.os.lower }} < 2.47.1, updating to >= 2.47.1 via a [self-service update][self-service-update] will enable delta updates for the device.
+For any devices running {{ $names.os.lower }} >= 2.47.1, the delta update behavior is enabled by default and can't be disabled. For devices running {{ $names.os.lower }} < 2.47.1, updating to >= 2.47.1 via a [self-service update][self-service-update] will enable delta updates for the device.
```

Comment:
@vipulgupta2048 I thought deltas v2 (rsync) would still be enabled on devices < 2.47.1?

Reply:
We don't specifically mention those devices above, do we? But, to answer your question. Yes, deltas are enabled by default on all devices if IIRC. It's just that they can't be disabled anymore is what I am concluding above.

Reply:
Ah, they's disabled by default <2.47.1 but you can enable them to receive rsync v2 deltas, is that what you're saying @vipulgupta2048 ?

Reply:
No @cywang117 What I am trying to say is I don't know and I am trying to conclude and document above what is correct. Here's what I think is correct:

- We always enabled the deltas, everywhere in all versions of balenaOS and in everything
- Below 2.47.1, folks had an option to disable the deltas if they wanted to 
- After 2.47.1, the option disappeared and they needed to do some digging to disable delta (We no longer surface the config variable in the UI)
- Present day: There is no way you can disable deltas. 

How am I doing @cywang117 For context, the PR originated from this comment by a user https://github.com/balena-io/docs/pull/1681#issuecomment-977961808 and I am trying to fix the language here

Reply:
That makes sense @vipulgupta2048 . Thanks for clarifying!

Reply:
Roger that, resolving. Do approve the PR.

Reply:
@cywang117 Check the latest commit out now, I have turned around the paragraph to now treat Delta as a first-class feature rather than something that is optional or secondary. This way, people won't assume it is something to be disabled or can be disabled.

Reply:
@vipulgupta2048 This LGTM. Self-certify if you feel confident about it :)